lib/internal/histogram.js

```diff
@@ -264,8 +268,12 @@ class Histogram {
 }
 
 class RecordableHistogram extends Histogram {
-  constructor() {
-    throw new ERR_ILLEGAL_CONSTRUCTOR();
+  constructor(skipThrowSymbol = undefined) {
+    if (skipThrowSymbol !== kSkipThrow) {
+      throw new ERR_ILLEGAL_CONSTRUCTOR();
+    }
```

Comment:
Couldn't the constructor be omitted entirely at this point, delegating responsibility for that throw to the parent class?

Reply:
If we leave it to the parent to throw the error, the stack trace will be different from one release to another.

Reply:
Ah, seems reasonable then. 👍🏻